README.rst

```diff
@@ -8,3 +8,38 @@ simply clone this repository and hack away at the scripts provided.
 .. code-block:: bash
 
    $ git clone git@github.com:mila-udem/blocks-examples.git
+
+
+Running the examples
+---------------------
+
+To pick the simplest example first, one this repository has been downloaded, 
+``cd`` into it, and ::
+
+    python -m sqrt --num-batches 1000 sqrt/saved_state
+
+
+Technical Note
+................
+
+Executing this command line will run the code in the ``sqrt`` 
+module : specifically the command-line argument parser in 
+the ``sqrt/__main__.py`` file.  Having the main entry point here is 
+mostly a quirk of the Python module system - we wanted to keep the 
+launching command as simple as possible.
+
+
+Understanding the examples
+---------------------------
+The bulk of the ``blocks`` code consists of 'plain functions' that are 
```

Comment:
I am afraid I don't understand this paragraph. Does 'plain functions' refer to the ones with leading underscore in the name?